### ISSUE TITLE
Update solc to 0.4.24

### ISSUE THREAD

**Issue by dwalintukan** [trufflesuite/truffle#950]
- [ ] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

Please update the solc version to 0.4.24. 0.4.23 has some issues compiling with `keccak256`. 

## Steps to Reproduce

Please provide the shortest amount of steps to reproduce your issue.

## Expected Behavior

What you expected to happen.

## Actual Results

What actually happened. Please give examples and support it with screenshots, copied output or error messages.

## Environment

* Operating System: 
* Ethereum client:
* Truffle version (`truffle version`):
* node version (`node --version`):
* npm version (`npm --version`):

**Comment by cgewecke** [comment]
@dwalintukan We've merged PRs for this to develop and are planning to publish a patch release early next week, probably Monday (UTC -7).

Truffle's nightly build [darq-truffle](https://www.npmjs.com/package/darq-truffle) already contains this change and is available by running:
```shell
$ npm install -g darq-truffle
$ darq-truffle test --network ropsten # Example command
```

**Comment by cgewecke** [comment]
Published with 4.1.9. Thanks for opening @dwalintukan.
